### PR TITLE
Fix case when context is None

### LINKED ISSUE #150: picking_dispatch: Short dump of availability check when the stock is not available
Steps to reproduce

1.Create a material of type stockable material
2. Create a sales order
3. Confirm the sales order
4. Create a picking dispatch
5. Add the newly created delivery order to the picking
6. Save and press availability check
picking_dispatch_-_odoo

### PR DESCRIPTION
**Pull request by lbellier**
Fixes https://github.com/OCA/stock-logistics-warehouse/issues/150

### CODE REVIEW

File: stock_lock_lot/models/stock_production_lot.py [hunk 1]
```diff
@@ -14,9 +14,9 @@ class StockProductionLot(models.Model):
     _track = {
         'locked': {
             'stock_lock_lot.mt_lock_lot': lambda self, cr, uid, obj,
-            ctx=None: obj.locked,
+            ctx = None: obj.locked,
```
Comment (pedrobaeza) [inline stock_lock_lot/models/stock_production_lot.py]: Don't put spaces here. This is not PEP8

**Comment by clonedagain** [comment]
:+1:

**Comment by pedrobaeza** [comment]
:+1:

**Comment by laetitia-gangloff** [comment]
:+1:

**Pull request merged**